ledfx/api/virtuals_tools.py

```diff
@@ -30,6 +30,55 @@ async def get(self, virtual_id) -> web.Response:
         """
         return await self.request_success("info", f"Available tools: {TOOLS}")
 
+    async def post(self, virtual_id, request) -> web.Response:
```

Comment:
this breaks backward compatibility. I am not saying the original implementation by my self is any good, but I don't want to do this without eyes open.

I need thoughts from others on if this is okay to do.

We have no way really to know who is an existing user. 

It may be better to implement a new api endpoint for oneshot scaled to your vision for this feature and leave this one in its "crippled" form, still functional, but not expanded...

Reply:
Noting users may be myself and yourself and blades front end test hack....

Reply:
Nothing worse than finding out post upgrade when you need it your art project / DJ set stopped working and reading the manual later to discover a mutation.

Not sure if you have made your changes yet as I am looking from phone.

The fault here is mine as I established a bad first implementation.

Reply:
@goodfaiter we are going to ho with whatever new baseline you want to go with and live forward from there. So just pi g when this is what you want and I will merge.